modules/ossm-config-otel.adoc

```diff
@@ -99,41 +99,51 @@ spec:
 Once you verify that you can see traces, lower the `randomSamplingPercentage` value or set it to `default` to reduce the number of requests.
 ====
 
-. Deploy the `bookinfo` application in `bookinfo` namespace:
+. Create the `bookinfo` namespace by running the following command:
 +
 [source, terminal]
 ----
 $ oc create ns bookinfo
 ----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s):
+
+.. If you are using the `InPlace` update strategy and your `Istio` resource is named `default`, then run the following command:
 +
 [source, terminal]
 ----
-$ oc label ns <namespace_name> istio.io/rev= # <1>
+$ oc label ns bookinfo istio-injection=enabled
 ----
 +
+.. If you are using the `RevisionBased` update strategy, then run the following commands:
++
+... Find your `<revision-name>` by running the following command:
```

Comment:
If <revision-name> is supposed to refer to a user replaced value (which seems to be the case because it's surrounded by brackets>, just an FYI that we should only use underscores for these values:
```suggestion
... Find your `<revision_name>` by running the following command:
```

But in either case, it might be less confusing to use "revision name" in plain text here, especially since that's the phrasing used in the next substep